docs/TypeChecker.rst

```diff
@@ -737,10 +737,10 @@ checking problem::
 
 This constraint system generates the constraints "``T(f)`` ==Fn ``T0
 -> T1``" (for fresh variables ``T0`` and ``T1``), "``(T2, X)`` <c
-``T0``" (for fresh variable ``T2``) and "``T2 conforms to
+``T0``" (for fresh variable ``T2``) and "``T2`` conforms to
 ``FloatLiteralConvertible``". As part of the solution, after ``T0`` is
 replaced with ``(i : Int, s : String)``, the second of
-these constraints is broken down into "``T2 <c ``Int``" and "``X`` <c
+these constraints is broken down into "``T2`` <c ``Int``" and "``X`` <c
```

Comment:
@benlangmuir Thanks. Fixed.

@jrtibbetts ``<c`` is a conversion constraint. ``<C`` is a construction constraint.